src/main/java/com/appdirect/sdk/vendorFields/controller/VendorRequiredFieldController.java

```diff
@@ -118,4 +125,10 @@ public void initBinder(final WebDataBinder webdataBinder) {
         webdataBinder.registerCustomEditor(List.class, new LocaleConverter());
         webdataBinder.registerCustomEditor(Locale.class, new LocaleObjectConverter());
     }
+
+    private Map<String, String> getAdditionalInformation(Map<String, String> headers) {
+        return new HashMap<String, String>() {{
+                put(HttpHeaders.X_FORWARDED_FOR, headers.get(HttpHeaders.X_FORWARDED_FOR));
+        }};
```

Comment:
Might be able to do this.
```suggestion
        return new Collections.singletonMap(HttpHeaders.X_FORWARDED_FOR, headers.get(HttpHeaders.X_FORWARDED_FOR));
```

Reply:
I try to not use Collections.singletonMap or Collections.of because there are limits on how many entries can be added. put is easier add additional entries